eng/Version.Details.xml

```diff
@@ -1,139 +1,139 @@
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="VS.Tools.Net.Core.SDK.Resolver" Version="9.0.100-preview.5.24304.3">
+    <Dependency Name="VS.Tools.Net.Core.SDK.Resolver" Version="9.0.100-preview.6.24312.9">
```

Comment:
Failing installing .NET dependencies:
```
D:\a\_work\1\s\src\DotNet\Dependencies\Workloads.csproj : error NU1101: Unable to find package Microsoft.NET.Sdk.Android.Manifest-9.0.100-preview.6. No packages exist with this id in source(s): benchmark-dotnet-prerelease, darc-pub-dotnet-runtime-919e5b2, dotnet-eng, dotnet-public, dotnet-tools, dotnet8, dotnet8-transport, dotnet9, dotnet9-transport, skiasharp [D:\a\_work\1\s\src\DotNet\DotNet.csproj]
D:\a\_work\1\s\src\DotNet\Dependencies\Workloads.csproj : error NU1101: Unable to find package Microsoft.NET.Sdk.MacCatalyst.Manifest-9.0.100-preview.6. No packages exist with this id in source(s): benchmark-dotnet-prerelease, darc-pub-dotnet-runtime-919e5b2, dotnet-eng, dotnet-public, dotnet-tools, dotnet8, dotnet8-transport, dotnet9, dotnet9-transport, skiasharp [D:\a\_work\1\s\src\DotNet\DotNet.csproj]
D:\a\_work\1\s\src\DotNet\Dependencies\Workloads.csproj : error NU1101: Unable to find package Microsoft.NET.Sdk.iOS.Manifest-9.0.100-preview.6. No packages exist with this id in source(s): benchmark-dotnet-prerelease, darc-pub-dotnet-runtime-919e5b2, dotnet-eng, dotnet-public, dotnet-tools, dotnet8, dotnet8-transport, dotnet9, dotnet9-transport, skiasharp [D:\a\_work\1\s\src\DotNet\DotNet.csproj]
D:\a\_work\1\s\src\DotNet\Dependencies\Workloads.csproj : error NU1101: Unable to find package Microsoft.NET.Sdk.tvOS.Manifest-9.0.100-preview.6. No packages exist with this id in source(s): benchmark-dotnet-prerelease, darc-pub-dotnet-runtime-919e5b2, dotnet-eng, dotnet-public, dotnet-tools, dotnet8, dotnet8-transport, dotnet9, dotnet9-transport, skiasharp [D:\a\_work\1\s\src\DotNet\DotNet.csproj]
D:\a\_work\1\s\src\DotNet\Dependencies\Workloads.csproj : error NU1101: Unable to find package Microsoft.NET.Sdk.macOS.Manifest-9.0.100-preview.6. No packages exist with this id in source(s): benchmark-dotnet-prerelease, darc-pub-dotnet-runtime-919e5b2, dotnet-eng, dotnet-public, dotnet-tools, dotnet8, dotnet8-transport, dotnet9, dotnet9-transport, skiasharp [D:\a\_work\1\s\src\DotNet\DotNet.csproj]
D:\a\_work\1\s\src\DotNet\DotNet.csproj(174,5): error MSB3073: The command ""D:\a\_work\1\s\bin/dotnet/dotnet" restore "D:\a\_work\1\s\src\DotNet\Dependencies/Workloads.csproj" -bl:D:\a\_work\1\s\artifacts\log\Release\/DotNetWorkloads.binlog" exited with code 1.
    6 Error(s)
```

Reply:
Due to an issue with ILLink we don't yet have builds with 9.0.100-preview.6 branded manifests, though I'm not sure where this naming requirement is coming from?